monitoring/templates/prometheus-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: prometheus
+  namespace: {{ .Release.Namespace }}
+  annotations:
+{{- if .Values.prometheusIngress.annotations }}
+{{ toYaml .Values.prometheusIngress.annotations | trim | indent 4 }}
+{{- end }}
+{{- if .Values.global.ingress.annotations }}
+{{ toYaml .Values.global.ingress.annotations | trim | indent 4 }}
+{{- end }}
+spec:
+  backend:
+    serviceName: urban-os-prometheus-server
```

Comment:
For the sake of cleanliness perhaps this could be called "monitoring-prometheus".  The urban-os nature is implied by it existing in the namespace

Reply:
good call, this has been changed.